OpenXmlPowerTools/ListItemTextGetter_de_DE.cs

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OpenXmlPowerTools
+{
+    public class ListItemTextGetter_de_DE
+    {
+        private static string[] OneThroughNineteen = {
+            "eins", "zwei", "drei", "vier", "fünf", "sechs", "sieben", "acht",
+            "neun", "zehn", "elf", "zwölf", "dreizehn", "vierzehn",
+            "fünfzehn", "sechzehn", "siebzehn", "achtzehn", "neunzehn"
+        };
+
+        private static string[] Tens = {
+            "zehn", "zwanzig", "dreißig", "vierzig", "fünfzig", "sechzig", "siebzig",
+            "achtzig", "neunzig"
+        };
+        
+        private static string[] OrdinalOneThroughNineteen = {
+            "erste", "zweite", "dritte", "vierte", "fünfte", "sechste",
+            "siebte", "achte", "neunte", "zehnte", "elfte", "zwölfte",
+            "dreizehnte", "vierzehnte", "fünfzehnte", "sechzehnte",
+            "siebzehnte", "achtzehnte", "neunzehnte"
+        };
+
+        private static string[] OrdinalTens = {
+            "zehnte", "zwanzigste", "dreißigste", "vierzigste", "fünfzigste",
+            "sechzigste", "siebzigste", "achtzigste", "neunzigste"
+        };
+
+        public static string GetListItemText(string languageCultureName, int levelNumber, string numFmt)
+        {
+			if (levelNumber > 19999)
+				throw new ArgumentOutOfRangeException("levelNumber", "Convering a levelNumber to ordinal text that is greater then 19 999 is not supported");
```

Comment:
Replace `Convering` with `Converting` in all of the language-related files of the commit. I recommend to write "19 999" as "19999" because the thausand separator is different in different languages (in English it is a `,`).

Reply:
Ok, done. I also found and fixed a few more typos in the code. Additionally, I added tests for German and Spanish.